src/main/java/ch/njol/skript/structures/StructCommand.java

```diff
@@ -259,6 +262,20 @@ public boolean load() {
 		}
 
 		String description = entryContainer.get("description", String.class, true);
+		String prefix = entryContainer.get("prefix", String.class, true);
+		for (char c : prefix.toCharArray()) {
+			if (Character.isWhitespace(c)) {
+				Skript.warning("command /" + command + " has a whitespace in its prefix. Defaulting to '" + DEFAULT_PREFIX + "'.");
+				prefix = DEFAULT_PREFIX;
```

Comment:
I'm pretty sure I said the opposite before but I changed my mind 😬 

I think ScriptCommand should allow `prefix` parameter to be null and just use `skript` in the constructor if the parameter is null

You could make `DEFAULT_PREFIX` field publicly accessible from there though (so you can use it in error messages here)